MapleServer2/Managers/Actors/Mob.cs

```diff
@@ -149,7 +151,7 @@ private void Move(MobMovement moveType)
                     }
 
                     NextMovementTarget = coordF;
-                    Distance = (coordF - Coord.ToShort()).Length();
+                    Distance = (coordF.ToFloat() - Coord).Length();
```

Comment:
isn't this `coordF` already a float?

Reply:
It was a lie. It was an coordS all the time